stable/artifactory/Chart.yaml

```diff
@@ -11,4 +13,6 @@ sources:
 maintainers:
   - name: Jainish shah
     email: jainishs@jfrog.com
+  - name: Eldad Assis
```

Comment:
Please use the Github username as name. We've started to recommend that because it helps identify the chart maintainer for future review.